modules/installation-creating-gcp-bootstrap.adoc

```diff
@@ -130,14 +130,14 @@ ifdef::shared-vpc[]
 +
 [source,terminal]
 ----
-$ gcloud compute instance-groups unmanaged add-instances ${INFRA_ID}-bootstrap-instance-group --zone=${ZONE_0} --instances=${INFRA_ID}-bootstrap
+$ gcloud compute instance-groups unmanaged add-instances ${INFRA_ID}-bootstrap-ig --zone=${ZONE_0} --instances=${INFRA_ID}-bootstrap
 ----
 
 . Add the bootstrap instance group to the internal load balancer backend service:
 +
 [source,terminal]
 ----
-$ gcloud compute backend-services add-backend ${INFRA_ID}-api-internal-backend-service --region=${REGION} --instance-group=${INFRA_ID}-bootstrap-instance-group --instance-group-zone=${ZONE_0}
+$ gcloud compute backend-services add-backend ${INFRA_ID}-api-internal-backend-service --region=${REGION} --instance-group=${INFRA_ID}-bootstrap-ig--instance-group-zone=${ZONE_0}
```

Comment:
Missing the space before `--instance-group-zone=${ZONE_0}`, please correct it, thanks!